doc/2/framework/classes/kuzzle-request/get-body-array/index.md

```diff
@@ -10,6 +10,7 @@ description: KuzzleRequest class getBodyArray() method
 <SinceBadge version="2.11.0" />
 
 Gets a parameter from the request body and checks that it is an array.
+We also support lodash syntax. [(```relations.lebron[0]```)](https://lodash.com/docs/4.17.15#get)
```

Comment:
Single line code formatting is done with two backtick 
```suggestion
We also support lodash syntax. [(`relations.lebron[0]`)](https://lodash.com/docs/4.17.15#get)
```
Can you please also correct the other occurrences please?

Reply:
done